gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.7-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.9-bin.zip
```

Comment:
This (and the gradle build tools) in a separate commit would also be appreciated, to keep build tool changes and features separate. (Helps when debugging compatibility issues.)

Reply:
Reverted.